template/analytics.notify.cve/body.html

```diff
@@ -58,7 +70,7 @@
         min-height: 300px;
         width: 100%;
         max-width: 800px;
-        margin: 0 auto;
+        margin: 100px auto;
```

Comment:
Is this needed?

Reply:
@invincibleJai, yep. otherwise, the two separations for transitive and direct dependencies had no space between them. Aesthetically, it looked bad.